libs/watt/package/menu/watt-menu.component.ts

```diff
@@ -119,7 +116,20 @@ export class WattMenuComponent {
    */
   hasIcons = computed(() => this.menuItems().some((item) => item.hasIcon()));
 
+  /**
+   * CSS classes to apply to the menu panel.
+   * @ignore
+   */
+  panelClasses = computed(() => {
+    const classes = ['watt-menu-panel'];
+    if (this.hasIcons()) {
+      classes.push('watt-menu-panel--has-icons');
+    }
+    return classes;
+  });
```

Comment:
suggestion: i think you can save some code by doing this in the template instead:

```html
<mat-menu
  class="watt-menu-panel"
  [class.watt-menu-panel--has-icons]="hasIcons()"
>
```